docs/reference/docs/multi-termvectors.asciidoc

```diff
@@ -1,14 +1,10 @@
 [[docs-multi-termvectors]]
-=== Multi termvectors API
+=== Multi term vectors API
```

Comment:
Not a big deal, but I think this would typically be hyphenated.

```suggestion
=== Multi-term vectors API
```

Reply:
It's actually multiple "term vectors" rather than multiple terms.